scripts/generate-coverage.sh

```diff
@@ -17,4 +17,4 @@ for d in $(go list ./... | grep -v vendor | grep -v tests); do
         cat profile.out >> coverage.txt
         rm profile.out
     fi
-done
\ No newline at end of file
+done
```

Comment:
Let's not add any space change to the original file.

Reply:
done